pkg/logging/kafka/update.go

```diff
@@ -64,6 +73,14 @@ func NewUpdateCommand(parent common.Registerer, globals *config.Data) *UpdateCom
 	c.CmdClause.Flag("format-version", "The version of the custom logging format used for the configured endpoint. Can be either 2 (default) or 1").Action(c.FormatVersion.Set).UintVar(&c.FormatVersion.Value)
 	c.CmdClause.Flag("placement", "Where in the generated VCL the logging call should be placed, overriding any format_version default. Can be none or waf_debug").Action(c.Placement.Set).StringVar(&c.Placement.Value)
 	c.CmdClause.Flag("response-condition", "The name of an existing condition in the configured endpoint, or leave blank to always execute").Action(c.ResponseCondition.Set).StringVar(&c.ResponseCondition.Value)
+	c.CmdClause.Flag("parse-log-keyvals", "Parse key-value pairs within the log format.").Action(c.ParseLogKeyvals.Set).BoolVar(&c.ParseLogKeyvals.Value)
+	c.CmdClause.Flag("no-parse-log-keyvals", "Disable parsing of key-value pairs within the log format.").Action(c.NoParseLogKeyvals.Set).BoolVar(&c.NoParseLogKeyvals.Value)
```

Comment:
We can collapse these two flags into a [`NegatableBool`](https://pkg.go.dev/gopkg.in/alecthomas/kingpin.v3-unstable#Clause.NegatableBool) which generates both variants of the bool for you `--parse-log-keyvals`, `--no-parse-log-keyvals`.

```suggestion
	c.CmdClause.Flag("parse-log-keyvals", "Parse key-value pairs within the log format.").Action(c.ParseLogKeyvals.Set).NegatableBool(&c.ParseLogKeyvals.Value)
```

Reply:
I just wanted to post this because it took me a while to find the docs on this since the library is not a direct import in this file

https://pkg.go.dev/gopkg.in/alecthomas/kingpin.v3-unstable#Clause.NegatableBool

Reply:
Done